DetectDynamicJS.py

```diff
@@ -154,15 +172,16 @@ def isScript(self, requestResponse):
             if len(contentTypeL) == 1:
                 contentType = contentTypeL[0].lower()
             statusCode = responseInfo.getStatusCode()
-            if (any(content in contentType for content in possibleContentTypes) or "script" in inferredMimeType or "script" in mimeType) and (int(statusCode) < 300 or int(statusCode) > 399):
-                return True
+            if (any(content in contentType for content in possibleContentTypes) or "script" in inferredMimeType or "script" in mimeType):
+              return True
```

Comment:
As soon as the right indendation is used, i'm happy to merge this. Nice additions!